src/components/atoms/bar-chart-content.tsx

```diff
@@ -196,36 +193,37 @@ const styles = StyleSheet.create({
     borderWidth: 1,
     borderColor: '#ACAFC4',
     position: 'absolute',
-    right: -10,
     minWidth: 60,
     borderRadius: 5,
     backgroundColor: 'white',
-    zIndex: 30
+    paddingBottom: 1
   },
   triangle: {
     width: 5,
-    height: 10,
+    height: 9.75,
     position: 'absolute',
-    bottom: -10,
+    bottom: -9.75,
     left: 36,
     borderLeftWidth: 7,
     borderLeftColor: 'transparent',
     borderRightWidth: 7,
     borderRightColor: 'transparent',
     borderTopWidth: 7,
-    borderTopColor: '#ACAFC4'
+    borderTopColor: '#ACAFC4',
+
   },
   triangle2: {
     width: 5,
-    height: 10,
+    height: 9.75,
```

Comment:
Sorry I should have been clearer: the previous comment only referred to the `bottom` value, setting it on height as well makes the triangle's line look very thin (unless you think it's necessary for a device you're testing on - in which case maybe let's just tolerate the hairline on some devices if trying to fix it makes it worse on others)

Reply:
That was the issue that I ran into, the original settings looked fine on my XR, but I'm guessing on smaller phones we are going to get the hairline border